backend/fms_core/containers.py

```diff
@@ -145,6 +146,14 @@ def __str__(self):
 
 # Run Containers
 
+CONTAINER_SPEC_AXIOM_96_FORMAT_ARRAY = ContainerSpec(
+    container_kind_id="axiom genetitan array",
```

Comment:
change the container kind id too (axiom 96-format array) (remember to change it in the template too).